postcss.config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  plugins: [
```

Comment:
I don't find these new packages in package.json. Is it built-in provided with React on Rails?